.github/workflows/build.yml

```diff
@@ -37,10 +37,17 @@ jobs:
           path: ~/.m2
           key: freelibrary-cache-${{ secrets.MAVEN_CACHE_KEY }}-${{ hashFiles('**/pom.xml') }}
           restore-keys: freelibrary-cache-${{ secrets.MAVEN_CACHE_KEY }}-
-      - name: Build with Maven
+      - name: Build with Maven, with required environment variables set
```

Comment:
Why "with required environmental variables set" here? This build doesn't use the optional-env-vars profile.

Reply:
I just figured more info = better, but happy to remove

Reply:
Nevermind me. I'm too tired and missed the "required" part of that. I need to call it a day.